cmd/medius/publish/docs.go

```diff
@@ -0,0 +1,67 @@
+package publish
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"kubevirt.io/containerdisks/pkg/quay"
+
+	"github.com/ghodss/yaml"
+	"kubevirt.io/containerdisks/cmd/medius/common"
+	"kubevirt.io/containerdisks/pkg/docs"
+)
+
+func NewPublishDocsCommand(options *common.Options) *cobra.Command {
+	options.PublishDocsOptions = common.PublishDocsOptions{
+		TokenFile: "",
+	}
+
+	publishCmd := &cobra.Command{
+		Use:   "docs",
+		Short: "Synchronize container disk descriptions with quay.io",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			success := true
+			client := quay.NewQuayClient(options.PublishDocsOptions.TokenFile, "containerdisks")
+			for _, p := range common.Registry {
+				if !p.UseForDocs {
+					continue
+				}
+				log := logger(p.Artifact)
+				metadata := p.Artifact.Metadata()
+				vm := docs.BasicVirtualMachine(metadata.Name, "quay.io/containerdisks/"+metadata.Name+":"+metadata.Version, metadata.ExampleCloudInitPayload)
+				example, err := yaml.Marshal(&vm)
```

Comment:
Is the `"quay.io/containerdisks/"` something which could be derived from the Quay module?

Reply:
I can make this nicer :+1: